### ISSUE TITLE
Cannot build TFLite 2.16.1 for RISC-V due to the bug in cpuinfo (missing sys/hwprobe.h)

### ISSUE THREAD

**Issue by eltimen** [tensorflow/tensorflow#64987]
### Issue type

Build/Install

### Have you reproduced the bug with TensorFlow Nightly?

Yes

### Source

source

### TensorFlow version

tflite 2.16.1

### Custom code

Yes

### OS platform and distribution

Linux

### GCC/compiler version

GCC 13.2.0 RISC-V GNU Compiler Toolchain link: https://github.com/riscv-collab/riscv-gnu-toolchain/releases/download/2023.11.17/riscv64-glibc-ubuntu-20.04-gcc-nightly-2023.11.17-nightly.tar.gz

### Current behavior?

I'm trying to build TensorFlow Lite 2.16.1 using CMake for RISC-V using GCC toolchain for this platform. The build fails with the following error inside `cpuinfo` dependency: 
```
[  0%] Building C object _deps/cpuinfo-build/CMakeFiles/cpuinfo.dir/src/riscv/linux/riscv-hw.c.o
build/cpuinfo/src/riscv/linux/riscv-hw.c:1:10: fatal error: sys/hwprobe.h: No such file or directory
      1 | #include <sys/hwprobe.h>
        |          ^~~~~~~~~~~~~~~
```
This bug is already fixed in cpuinfo by [this commit](https://github.com/pytorch/cpuinfo/commit/313524ab20d2041854af8ad07bf726ddd485d258).
Can you bump cpuinfo to the latest commit (or to any version including this fix) in the next TF release?

### Standalone code to reproduce the issue

```shell
cmake -S tensorflow/lite -B build -D CMAKE_BUILD_TYPE=Release -D CMAKE_SYSTEM_PROCESSOR=riscv64 -D BUILD_SHARED_LIBS=ON -D TFLITE_ENABLE_GPU=ON -D CMAKE_SYSTEM_NAME=Linux -D CMAKE_C_COMPILER=/opt/riscv_gcc/bin/riscv64-unknown-linux-gnu-gcc -D CMAKE_CXX_COMPILER=/opt/riscv_gcc/bin/riscv64-unknown-linux-gnu-g++ -D CMAKE_FIND_ROOT_PATH_MODE_PROGRAM=NEVER -D CMAKE_FIND_ROOT_PATH_MODE_LIBRARY=ONLY -D CMAKE_FIND_ROOT_PATH_MODE_INCLUDE=ONLY -D CMAKE_FIND_ROOT_PATH_MODE_PACKAGE=ONLY
```

**Comment by LakshmiKalaKadali** [comment]
Hi @eltimen,

The [commit ](https://github.com/pytorch/cpuinfo/commit/313524ab20d2041854af8ad07bf726ddd485d258) you have 
provided for cpuinfo was merged under pytorch  repo. But under TensorFlow repo (tensorflow/lite/tools/cmake/modules/cpuinfo.cmake), a [PR](https://github.com/tensorflow/tensorflow/pull/63850/files)  for `cpuinfo.cmake` still under open. Once it is reviewed and merged,  it will be updated on the next coming release.
Thank You

**Comment by eltimen** [comment]
Hi @LakshmiKalaKadali,
Thank you, I hadn't noticed that a PR to fix this had already been created.

**Comment by LakshmiKalaKadali** [comment]
Hi @eltimen,

Thank you for the response. We will update you once the PR #63850 is merged. Also please keep  track of the same [PR](https://github.com/tensorflow/tensorflow/pull/63850/files) .

Thank You

**Comment by NominBa** [comment]
hi All! 

tensorflow 2.16.1 - tflite build does not generate a static libraries in **/clog-build**. I see that the clog-build in the version r2.10. Do you have an any idea how can i fix it?

error output:

gmake[2]: *** No rule to make target '../../tflite_build/_deps/clog-build/libclog.a', needed by 'testapp/testapp'.  Stop.

**Comment by LakshmiKalaKadali** [comment]
Hi NominBa,

Could you please open a new issue.

Thank You
